using-mobile-extensions/adobe-target/target-api-reference.md

```diff
@@ -862,271 +1131,153 @@ var targetParameters = new ACPTargetParameters(mboxParameters, profileParameters
 
-let targetParameters = ACPTargetParameters(parameters: nil, profileParameters: nil, product: product, order: order)
+#### Syntax
 
-ACPTarget.locationsDisplayed(["mboxName1", "mboxName2"], with: targetParameters)
+```javascript
+setPreviewRestartDeeplink(deepLink: string)
 ```
-{% endtab %}
+* _deepLink_ is a string containing the preview restart deeplink.
 
-{% tab title="React Native" %}
-### locationsDisplayed
+#### Examples
 
 **JavaScript**
 
 ```javascript
-var product = new ACPTargetProduct("24D334", "Stationary");
-var order = new ACPTargetOrder("ADCKKBC", 400.50, ["34", "125"]);
-var targetParameters = new ACPTargetParameters(null, null, product, order);
-
-ACPTarget.locationsDisplayed(["mboxName1", "mboxName2"], targetParameters);
+ACPTarget.setPreviewRestartDeeplink("myapp://HomePage");
 ```
+
 {% endtab %}
-{% endtabs %}
 
-## Send an mbox click notification
+{% endtabs %}
 
-Use this API to send a location \(mbox\) click notification to configured Target server. This API can be invoked in the following cases:
+## setThirdPartyId
 
-* For a prefetched mbox, after the mbox content is retrieved using [retrieveLocationContent API](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-target/target-api-reference#retrieve-location-content-requests).
-* For a regular mbox when no previous prefetch request is made and the mbox content is retrieved using `retrieveLocationContent` API.
+This API sets the custom visitor ID for Target.
 
-{% hint style="warning" %}
-For a click notification to be sent to Target, make sure click metric is enabled for the given mbox name in Target.
+{% hint style="info" %}
+This ID is preserved between app upgrades, is saved and restored during the standard application backup process, and is removed at uninstall or when the `resetExperience` API is used.
 {% endhint %}
 
 {% tabs %}
 {% tab title="Android" %}
-### locationClicked
 
-#### Syntax
+### Syntax
 
 ```java
-public static void locationClicked(final String mboxName, final TargetParameters parameters)
+public static void setThirdPartyId(final String thirdPartyId)
 ```
 
-#### Example
-
-```java
-// Mbox parameters
-Map<String, Object> mboxParameters = new HashMap<>();
-mboxParameters.put("membership", "prime");
-
-// Product parameters
-Map<String, Object> productParameters = new HashMap<>();
-productParameters.put("id", "CEDFJC");
-productParameters.put("categoryId","Electronics");
-
-List<String> purchasedIds = new ArrayList<String>();
-purchasedIds.add("81");
-purchasedIds.add("123");
-purchasedIds.add("190");
-
-// Order parameters
-Map<String, Object> orderParameters = new HashMap<>();
-orderParameters.put("id", "NJJICK");
-orderParameters.put("total", "650");
-orderParameters.put("purchasedProductIds",  purchasedIds);
-
-// Profile parameters
-Map<String, Object> profileParameters = new HashMap<>();
-profileParameters.put("ageGroup", "20-32");
+* _thirdPartyId_ is a String containing the custom visitor ID to be set in Target.
```

Comment:
Please reword as follows:
"_thirdPartyId_ is a string that contains the custom visitor ID to be set in Target.